woocommerce/woocommerce-gateway-stripe
# Tweak: select full text on focus in the Stripe key text boxes

## Changes proposed in this Pull Request:
The Stripe keys (publishable key, secret key, webhook secret) are not supposed to be edited by single characters and are generally removed or replaced as a whole string. This PR is a small tweak to select the full key string when the text boxes are clicked.

## Testing instructions
- Go to the Stripe settings page.
- Click the `Edit Stripe Keys` button to open the modal.
- Click on the publishable key/ secret key/ webhook secret text boxes. The full key should be selected on click.

**Before**
![before](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/4f6e811d-602e-4b3b-9fe4-7f53392190c3)

**After**
![now](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/72d65248-ae9f-4092-a5f2-5d14f764b282)
